alpha/lib/model/TrackEntryPeer.php

```diff
@@ -10,6 +10,8 @@
  */ 
 class TrackEntryPeer extends BaseTrackEntryPeer implements IRelatedObjectPeer
 {
+	const PEER_GET_LIMIT = 30;
```

Comment:
thought about that at first as well but eventually moved it to the peer class... will move it back to the service again.